packages/wallet/src/redux/protocols/existing-channel-funding/readme.md

```diff
@@ -0,0 +1,74 @@
+# Existing Channel Funding Protocol
+
+The purpose of this protocol is to fund a channel using an existing ledger channel. The protocol should
+
+- Check if a ledger channel has enough funds to fund the requested channel.
+  - If not it should initiate a ledger top-up protocol.
+- Craft ledger updates to fund the channel. (Perhaps we should extract this into a protocol that can be shared by indirect-funding and existing channel funding.)
+
+Currently we assume a Ledger Top-up protocol handles both the cases where a current player needs to top-up funds and the case where their opponent needs to top-up.
+
+# State Machine
+
+### Player A State Machine
```

Comment:
I think it's sad that this will almost immediately need to be refactored to account for two-or-three-party channels.